framework/docs/content/plugin-platform.md

```
@@ -126,6 +129,124 @@ re-filters against the same allowlist at serve time, mirroring how
+	pluginhost.WithWorkerCSP(pluginhost.WorkerCSP{
+		ScriptKeywords: []string{"'unsafe-eval'"},
+		ConnectSources: []string{"'self'"},
+		WASM:           true,
+	}),
+	pluginhost.WithCache(pluginhost.CachePrivateNoStore),
+)
+```
+
+The worker policy is a fixed skeleton plus validated tokens:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)